### ISSUE TITLE
Bug in scaleLog?

### ISSUE THREAD

**Issue by DaveHaasPowderSoftware** [d3/d3-scale#126]
Hi.

I think the readme may be incorrect. In the current implementation of scaleLog I cannot see how the base is taken into account in the deinterpolate method.

The readme says that the default base is 10, but as far as I can tell it's LN (2.718) evidenced by the calls to Math.log in deinterpolate.

Also, in playing with the code calling scale.base(...) has no effect, unless I'm doing something wrong.

Basically I do:

var logScale = d3.scaleLog().domain([1, 100]).range([200, 500]);
console.log(logScale(67));

logScale.base(2.718);

console.log(logScale(67));

I'm expecting the 2 calls to produce different results but they don't (they both produce 473.911). Am I missing something?

Dave

**Comment by DaveHaasPowderSoftware** [comment]
I found the solution. Changing the base only changes the placement of ticks. It has no effect on the actual scale.

Sorry about that.

**Comment by inferiorhumanorgans** [comment]
The way the README is written it does indeed appear that the base should be taken into account.  IMO this should be changed in the code or documentation.
